app/src/behaviors/behavior_sleep.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#define DT_DRV_COMPAT zmk_behavior_sleep
+
+#include <device.h>
+#include <drivers/behavior.h>
+#include <logging/log.h>
+
+#include <zmk/behavior.h>
+#include <zmk/activity.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+static int behavior_sleep_init(const struct device *dev) { return 0; };
+
+static int on_keymap_binding_released(struct zmk_behavior_binding *binding,
+                                      struct zmk_behavior_binding_event event) {
+    return activity_set_state(ZMK_ACTIVITY_SLEEP);
+}
+
+static const struct behavior_driver_api behavior_sleep_driver_api = {
+    .binding_released = on_keymap_binding_released,
+};
```

Comment:
`.binding_pressed` should be defined so you don't get an error in logs when pressing this key. You can have a function that just returns an opaque response like this: https://github.com/zmkfirmware/zmk/blob/main/app/src/behaviors/behavior_rgb_underglow.c#L54

Reply:
Added